src/redux/categories/categories.js

```diff
@@ -0,0 +1,14 @@
+export function checkStatus() {
+  return {
+    type: 'CHECK_STATUS',
+  };
+}
+
+export default function categoriesReducer(categories = [], action) {
+  switch (action.type) {
+    case 'CHECK_STATUS':
+      return 'Under Construction';
+    default:
+      return categories;
+  }
+}
```

Comment:
- Hey, the same issue I've pointed out in the reviews above is applicable here in these `categories`, Please, fix them.

Reply:
Thank you, sir, for raising my attention to these mistakes.